taskcluster/scripts/fetch/enter_dev_shell.ps1

```diff
@@ -28,8 +28,8 @@ ForEach-Object -InputObject $INCLUDE_ADDS {
 }
 
 $LIB_ADDS = `
-    "$W10_SDK_PATH\lib\$W10_SDK_VERSION\ucrt\x64;" ,`
-    "$W10_SDK_PATH\lib\$W10_SDK_VERSION\um\x64;" ,`
+    "$W10_SDK_PATH\Lib\$W10_SDK_VERSION\ucrt\x64;" ,`
+    "$W10_SDK_PATH\Lib\$W10_SDK_VERSION\um\x64;" ,`
```

Comment:
What's this about?

Reply:
This script set's up the env for windows on taskcluster. Normally a properly installed vs-studio does that, but we can't have nice things. 
So we need to set up the paths for the linker manually. Those 2 were broken but nobody noticed because we did not use any lib from those folders until now :)

Reply:
Got it, thanks!